backend/src/bin/genereate.ts

```diff
@@ -0,0 +1,17 @@
+import * as fs from "node:fs"
+
+import app from "../app"
+
+const schema = app.getOpenAPI31Document({
+  info: {
+    license: { name: "" },
+    title: "Problem and Submission API",
+    version: "0.1.0",
+  },
+  openapi: "3.1.0",
+})
```

Comment:
DRY原則に基づき、`src/app.ts`にて定義されているopenApiDocumentを使った方が良いと思います！

```suggestion
import app, { openApiDocument } from "../app"

const schema = app.getOpenAPI31Document(openApiDocument)
```